java/src/main/java/ai/pluggy/quickstart/QuickstartApplication.java

```diff
@@ -0,0 +1,122 @@
+      System.out.println(MessageFormat.format("(#{0}) - {1}", connector.getId(), connector.getName()));
+    });
+
+    // Fetch Pluggy Bank Connector
+    Connector connector = pluggyClient.service()
+        .getConnector(pluggyBankConnectorId)
+        .execute()
+        .body();
+    System.out.println(MessageFormat.format("We are going to connect with {0}", connector.getName()));
+
+    System.out.println("We will send the sucess case login parameters.");
+    System.out
+        .println(MessageFormat.format("User \"{0}\" with password \"{1}\"", pluggyBankUser, pluggyBankPassword));
+
+    // Create Item
+    ParametersMap parameters = new ParametersMap();
+    parameters.put("user", pluggyBankUser);
+    parameters.put("password", pluggyBankPassword);
+    CreateItemRequest createItemRequest = new CreateItemRequest(pluggyBankConnectorId, parameters, webhookUrl);
+
+    Response<ItemResponse> itemResponse = pluggyClient.service().createItem(createItemRequest).execute();
+    if (!itemResponse.isSuccessful()) {
+      System.out.println("There was a problem while creating the item.");
+      System.out.println(itemResponse.errorBody().string());
+      System.exit(1);
+    }
+
+    ItemResponse item = itemResponse.body();
+    String id = item.getId();
+
+    String status = item.getStatus();
+    while (!status.equals("LOGIN_ERROR") && !status.equals("OUTDATED") && !status.equals("UPDATED")) {
+      System.out.println(MessageFormat.format("Item {0} is syncing with the institution", id));
+      // sleep
+      Thread.sleep(2000);
+      itemResponse = pluggyClient.service().getItem(id).execute();
+      item = itemResponse.body();
+      status = item.getStatus();
+    }
+
+    System.out.println(MessageFormat.format("Item completed execution with status {0}", status));
+    if (status.equals("LOGIN_ERROR") || status.equals("OUTDATED")) {
+      System.exit(1);
+    }
+
+    // You can skip to this part
+    // if you are using Pluggy Connect and just use the item ID directly
+    // Retrieve item data
+    System.out.println(MessageFormat.format("Retrieving accounts for item {0}", id));
+    List<Account> accounts = pluggyClient.service().getAccounts(id).execute().body().getResults();
+    accounts.forEach(account -> {
+      String accountId = account.getId();
+      System.out.println(MessageFormat.format("Account {0} with balance {1} and number {2}", accountId,
+          account.getBalance(), account.getNumber()));
+
+      try {
+        List<Transaction> transactions = pluggyClient.service().getTransactions(accountId).execute().body()
+            .getResults();
+
+        transactions.forEach(transaction -> {
+          System.out.println(MessageFormat.format("Transaction {0} made at {1}, description \"{2}\", amount {3}",
+              transaction.getId(), transaction.getDate(), transaction.getDescription(), transaction.getAmount()));
+        });
+      } catch (IOException e) {
+        e.printStackTrace();
```

Comment:
I'd log a more contextualized error message

Reply:
Like what? If it were for me I wouldn't even catch the error since this success case is meant for showcase only, but is mandatory for Java to catch it because is surrounded in a lambda handler